avm/res/container-service/managed-cluster/main.bicep

```diff
@@ -131,7 +131,7 @@ param aadProfileEnableAzureRBAC bool = enableRBAC
 @description('Optional. If set to true, getting static credentials will be disabled for this cluster. This must only be used on Managed Clusters that are AAD enabled.')
 param disableLocalAccounts bool = true
 
-@description('Optional. Node provisioning settings that apply to the whole cluster.')
+@description('Optional. AUTO MODE IS A PARAMETER USED FOR A PREVIEW FEATURE, MICROSOFT MAY NOT PROVIDE SUPPORT FOR THIS, PLEASE CHECK THE PRODUCT DOCS (https://learn.microsoft.com/en-us/azure/aks/learn/quick-kubernetes-automatic-deploy?pivots=bicep#before-you-begin) FOR CLARIFICATION. Node provisioning settings that apply to the whole cluster.')
```

Comment:
Do we really need this screaming caps error message in so many place?

Reply:
> Do we really need this screaming caps error message in so many place?

@jongio - This is required in the development documentation [spec](https://azure.github.io/Azure-Verified-Modules/specs/shared/#id-sfr1---category-composition---preview-services).

@AlexanderSehr - Do you have any ideas?

Reply:
Hey @jongio,
this is kind of the idea - screaming at the user as it is a preview feature. It is certainly not pretty no - and truth be told I'm not the biggest fan myself. But the intent behind the spec makes sense. 
@matebarabas / @jtracey93 I guess there's not really a short handle, is there?

One small change I would suggest though @zedy-wj would be to change the text to `Optional. AUTO MODE IS A PARAMETER USED FOR A PREVIEW FEATURE, MICROSOFT MAY NOT PROVIDE SUPPORT FOR THIS, PLEASE CHECK THE [PRODUCT DOCS](https://learn.microsoft.com/en-us/azure/aks/learn/quick-kubernetes-automatic-deploy?pivots=bicep#before-you-begin) FOR CLARIFICATION. Node provisioning settings that apply to the whole cluster.`

Which would render the URL as a hyperlink.

Reply:
@AlexanderSehr - Updated.

Reply:
@AlexanderSehr, I like the idea of adding the link to the related docs.

Reply:
@jongio are we good?

Reply:
@jongio can you review the updates above and unblock?